src/computed/channel.js

```diff
@@ -75,6 +75,16 @@ const ComputedChannel = store => {
       const { channelBalanceClosingSatoshis, settings } = store;
       return toAmountLabel(channelBalanceClosingSatoshis, settings);
     },
+    get channelBalanceForceClosingSatoshis() {
+      return (store.pendingChannels || [])
+        .filter(c => c.status === 'pending-force-closing')
+        .map(c => c.localBalance)
+        .reduce((a, b) => a + b, 0);
+    },
+    get channelBalanceForceClosingLabel() {
```

Comment:
Yeah. I just added it to be consistent with the other computed values. Mainly because we test the labels directly in the unit tests and not the satoshi values. I'd suggest keeping the function in case we need it in a future design iteration.